.gitignore

```diff
@@ -38,6 +38,7 @@ x64/
 *.vspscc
 *.vssscc
 .builds
+GeneratedHelpAboutVersion.cs
```

Comment:
Excluding the generated file, so we can reliably generate the file on the build machine.